lib/styles/styles.dart

```diff
@@ -133,10 +135,28 @@ class _Text {
 
 @immutable
 class _Times {
-  final Duration fast = Duration(milliseconds: 300);
-  final Duration med = Duration(milliseconds: 600);
-  final Duration slow = Duration(milliseconds: 900);
-  final Duration pageTransition = Duration(milliseconds: 200);
+  _Times(this.disabled);
+  final bool disabled;
+  late final Duration xxxFast = Duration(milliseconds: disabled ? 1 : 50);
+  late final Duration xxFast = Duration(milliseconds: disabled ? 1 : 100);
+  late final Duration xFast = Duration(milliseconds: disabled ? 1 : 150);
+  late final Duration fast = Duration(milliseconds: disabled ? 1 : 300);
+  late final Duration medFast = Duration(milliseconds: disabled ? 1 : 450);
+  late final Duration med = Duration(milliseconds: disabled ? 1 : 600);
+  late final Duration slow = Duration(milliseconds: disabled ? 1 : 900);
+  late final Duration xSlow = Duration(milliseconds: disabled ? 1 : 1200);
+  late final Duration xxSlow = Duration(milliseconds: disabled ? 1 : 1500);
+  late final Duration xxxSlow = Duration(milliseconds: disabled ? 1 : 1800);
+  late final Duration pageTransition = Duration(milliseconds: disabled ? 1 : 200);
+  late final Duration introPageTransition = Duration(milliseconds: disabled ? 1 : 250);
+  late final Duration zoom = Duration(milliseconds: disabled ? 1 : 1350);
+  late final Duration ribbon = Duration(milliseconds: disabled ? 1 : 400);
+  late final Duration swipeDuration = Duration(milliseconds: disabled ? 1 : 240);
+  late final Duration delayFast = Duration(milliseconds: disabled ? 0 : 250);
+  late final Duration delayMed = Duration(milliseconds: disabled ? 0 : 500);
+  late final Duration delaySlow = Duration(milliseconds: disabled ? 0 : 700);
+  late final Duration delayXSlow = Duration(milliseconds: disabled ? 0 : 1050);
+  late final Duration delayXXSlow = Duration(milliseconds: disabled ? 0 : 4000);
```

Comment:
I don't really want to add a ton of extra fields here... I think for most of these we will want to keep them as "non standard" values inside the view and we'll just support low-motion directly in that view. But I don't want to add a ton of boilerplate either. I'll have to think on this for a bit...